src/core/ext/xds/xds_api.cc

```diff
@@ -882,7 +882,7 @@ absl::StatusOr<XdsApi::ResourceName> ParseResourceNameInternal(
       uri->query_parameter_map().begin(), uri->query_parameter_map().end());
   std::sort(query_parameters.begin(), query_parameters.end());
   return XdsApi::ResourceName{
-      uri->authority(),
+      absl::StrCat("xdstp:", uri->authority()),
       absl::StrCat(
           "xdstp:", path_parts.second, (query_parameters.empty() ? "?" : ""),
```

Comment:
Similarly, we no longer need the `"xdstp:"` here, since it's now in the authority.

Reply:
done